ocf-servers/js-servers/environmental_sensor.js

```diff
@@ -18,17 +18,19 @@ var	device = require('iotivity-node')('server'),
 	sensorData =   {temperature : 0.0,
 			humidity : 0.0,
 			pressure : 0.0,
-			uvIndex : 0};
+			uvIndex : 0},
+    simData = 0.0;
 
 var noble = '';
 try {
 	noble = require('noble');
 }
 catch(e) {
 	debuglog('Require noble module: ', e.message);
```

Comment:
One suggestion from me here is to add a note in the code (and possibly in debuglog too) that we are nevertheless proceeding but that all data sent is fake data.